src/include/sof/schedule/schedule.h

```diff
@@ -50,7 +51,6 @@ enum {
 struct scheduler_ops {
 	void (*schedule_task)(void *data, struct task *task, uint64_t start,
 			      uint64_t period);
-	int (*schedule_task_init)(void *data, struct task *task);
```

Comment:
@tlauda this change doesnt look complete. If you remove the init from the ops, the free needs to go too right?

Reply:
No, free can still be abstracted.